pkg/redeemertesting/redeem_test.go

```diff
@@ -225,7 +225,47 @@ func (s *redeemTestSuite) TestTempusRedeem() {
 }
 
 func (s *redeemTestSuite) TestIlluminateRedeem() {
+	assert := assert.New(s.T())
+
+	amount := big.NewInt(1000)
+	maturity := big.NewInt(9999999)
+	owner := common.HexToAddress("0x0000000000000000000000000000000000000002")
```

Comment:
remember that you have `s.Env.User1` and `s.Env.User2` for what should be, well, _users_